data/korath/korath ships.txt

```diff
@@ -384,6 +384,7 @@ ship "'nra'ret"
 ship "'olofez"
 	plural "'olofezes"
 	sprite "ship/chaser"
+		"scale" 0.6
```

Comment:
How are we going to handle bay counts ? 

At a scale of 0.7, I would increase a Palavet's compliment of 'olofez to 4

At a scale of 0.6, as you have, I would give it 4 'olofez and 2 drones.  I think this would a a reasonable increase.

At a scale of 0.5, things get *fun* and we can get *experimental*.  Now you can stack them in all three dimensions.  In theory you could cram 16 'olofez on a Palavret, but to keep things reasonable sane and provide space for a hot landing, I'd have 6 'olofez plus 4 drones.

The KWS-E becomes a serious carrier at that point, too.

Reply:
I wasn't planning on changing the bay counts within this PR, just keeping it focused on the scaling alone.

But yes, I fully support expanding the counts of carried ships on any ship where bays and carried things are an important feature. Ex, I'm not going to expand the bays on the Lampyrid, because both lore says 2, and because bays are not a core aspect of its identity. 

I should also note that my initial pass for scaling put fighters at 0.6, but subsequent testing and updating the hardpoints looks better to have them at 0.5. 

So I'm currently running on the plan that I'm going to be adjusting that to 0.5 as I get to them. Or hopefully, as other people get to them. Which is why I adjusted the title of this PR and description to clarify this.

Reply:
Ah, nice !  And yes the Lampyrid definitely needs a pair of shuttles :sunglasses: It's kinda too small to fit normally-sized shuttles anyway.

Add "make a carryable shuttle for the Lampyrid in blender" to my to-do list lol

Reply:
I figured that the drop ship was probably a close enough analog to the little shuttles from Firefly.